downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -227,9 +235,10 @@ public type SecureSocket record {|
 
 - Changed the SSLMode value from `SSL_VERIFY_CERT` to `SSL_VERIFY_CA`.
 
-##### Xmldata Package
+##### XML Data Package Updates
```

Comment:
Check if XmlData is the package name. If it is - Updates to the xmlData Package (or something like that). I'm not sure, so please check.

Reply:
Same with the other package updates.

Reply:
@kalaiyarasiganeshalingam

Reply:
@praneesha , @dakshitha Let's use the package name `XmlData` as we can't split the package name?

Reply:
Updated. Thanks a lot!